src/mon/OSDMonitor.cc

```diff
@@ -7652,28 +7755,28 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
     }
     if (prefix == "osd crush weight-set create") {
       if (osdmap.require_min_compat_client > 0 &&
-	  osdmap.require_min_compat_client < CEPH_RELEASE_LUMINOUS) {
-	ss << "require_min_compat_client "
-	   << ceph_release_name(osdmap.require_min_compat_client)
-	   << " < luminous, which is required for per-pool weight-sets. "
+	      osdmap.require_min_compat_client < CEPH_RELEASE_LUMINOUS) {
+	    ss << "require_min_compat_client "
+	       << ceph_release_name(osdmap.require_min_compat_client)
+	       << " < luminous, which is required for per-pool weight-sets. "
            << "Try 'ceph osd set-require-min-compat-client luminous' "
            << "before using the new interface";
-	err = -EPERM;
-	goto reply;
+	    err = -EPERM;
+	    goto reply;
       }
       string poolname, mode;
       cmd_getval(cct, cmdmap, "pool", poolname);
       pool = osdmap.lookup_pg_pool_name(poolname.c_str());
       if (pool < 0) {
-	ss << "pool '" << poolname << "' not found";
-	err = -ENOENT;
-	goto reply;
+	    ss << "pool '" << poolname << "' not found";
+	    err = -ENOENT;
+	    goto reply;
```

Comment:
sorry for nitpicking whitespace, but . . . doesn't the C++ code use two-space indentation everywhere? (So, in this case, shouldn't we fixing the single-space indentation by just adding another space to make it two?)

Reply:
I'm pretty sure that what's showing up on GitHub isn't matching what's there... (maybe there are mixed spaces or tabs or something?).

Reply:
@chardan I looked at the passages in current master and they appear to be properly indented. Then I compared with your patch and it seems this patch does introduce whitespace changes - 4-space indents instead of 2-space, etc.

So, I'm thinking it should be just a matter of going over the changed lines in this patch . . . and if the change is whitespace-only, revert it?